### ISSUE TITLE
_tkinter.TclError: couldn't recognize data in image file "C:\Python27\lib\site-packages\tkfilebrowser\images\file.png"

### ISSUE THREAD

**Issue by fxshpac** [j4321/tkFileBrowser#11]
I got the error in the title when running the following code based on what's from https://pypi.org/project/tkfilebrowser/. I have installed latest python 2.7.15 on Windows 10 which ships with Tkinter version 8.5 and I heard that version 8.6 starts supporting png natively which indicates 8.5 may have issue with png? Just guessing. But since Python 2.7 is supported version I figure I will bring up this issue anyway.

```
from Tkinter import *
import Tkinter, Tkconstants, tkFileDialog
from tkfilebrowser import askopendirname, askopenfilenames, asksaveasfilename

root = Tk()
filename = askopenfilenames(parent=root, filetypes=[("JPEG", "*.jpg"), ("All files", "*")])
print (filename)
root.destroy()
```

```
Python 2.7.15 (v2.7.15:ca079a3ea3, Apr 30 2018, 16:30:26) [MSC v.1500 64 bit (AMD64)] on win32
>>> Tkinter.TkVersion
8.5
```

**Comment by j4321** [comment]
Thanks for reporting. This is indeed a compatibility issue with tk 8.5 that does not support PNG files. I am using Linux and I had not checked the tk version shipped with python 2.7 for Windows. 

However, as written in the description, this module is targeting linux platform only and the tk version currently available in the main distributions like Ubuntu is 8.6. I have never tested my code in Windows and there are probably be other issues as well. In my opinion, the tkinter.filedialog module provides usable file-browser dialogs in Windows, while there are very old looking and unpractical in linux, which is the reason why I created tkfilebrowser. 

The compatibility issue is easy enough to fix but it will add dependencies unnecessary to most users so I will address it only if a Linux user asks for tk 8.5 compatibility. 

Let me know if you would be interested in a Windows version, knowing that the dialogs are meant to look like GTK+ dialogs in XFCE. For instance, the left pane is not a treeview but a shortcut list:
![capture d ecran_2018-08-07_13-16-59](https://user-images.githubusercontent.com/18469869/43772839-cb44f2a8-9a43-11e8-8989-00e96080fade.png)

**Comment by fxshpac** [comment]
@j4321 Thx for your detailed response. Actually you can close it since tkinter.filedialog module provides usable file-browser dialogs in Windows. I was just curious to see how yours can provide better features.
